tests/python/rest_api/test_jobs.py

```diff
@@ -510,3 +515,87 @@ def test_can_export_annotations(self, admin_user: str, jobs_with_shapes: List):
         job = jobs_with_shapes[0]
         response = self._export_annotations(admin_user, job["id"], format="CVAT for images 1.1")
         assert response.data
+
+    @pytest.mark.parametrize("username, jid", [("admin1", 14)])
+    def test_check_exported_coco_annotations_structure(
+        self,
+        username,
+        jid,
+        tasks,
+        jobs,
+        annotations,
+    ):
+        job_data = jobs[jid]
+        annotations_before = annotations["job"][str(jid)]
+        task_id = job_data["task_id"]
+        task_size = tasks[task_id]["size"]
+
+        response = self._export_annotations(username, jid, format="COCO 1.0")
+        assert response.data
+        temp_anno_file_name = None
+        with NamedTemporaryFile(
+            mode="w+b", prefix="cvat", delete=False, suffix=".zip"
+        ) as temp_anno_file:
+            temp_anno_file_name = temp_anno_file.name
+            temp_anno_file.write(response.data)
```

Comment:
What is the purpose of actual writing files on disk? Probably, it will be better to write the data to `BytesIO` or directly to `ZipFile`.